code/datums/components/squeak.dm

```diff
@@ -72,6 +72,8 @@
 		steps++
 
 /datum/component/squeak/proc/play_squeak_crossed(atom/source, atom/movable/crossing)
+	if(!isturf(crossing.loc))
```

Comment:
Any specific reason to not use the parent (aka the item that does the squeaking) here? I think it would be more consistent
```suggestion
	if(!isturf(parent.loc))
```

Reply:
Hm, maybe I should do both. If it's just parent/source then standing over a ducky and putting sommething into your backpack would still squeak. Walking over a backpack with a ducky inside doesn't squeak in the current iteration, so it might not be necessary to test both. I honestly don't fully understand why that's the case without further probing. Should I check for both just in case?

Reply:
I pushed a commit to check for both and re tested it.